app/views/shared/_powered.haml

```diff
@@ -0,0 +1,3 @@
+%li
+  powered by 
+  = link_to 'Hackerspace.gr', "https://www.hackerspace.gr", :target => "_blank"
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Use the new Ruby 1.9 hash syntax.